crates/protocol/interop/src/graph.rs

```diff
@@ -13,6 +13,9 @@ use kona_genesis::RollupConfig;
 use kona_registry::{HashMap, ROLLUP_CONFIGS};
 use tracing::{info, warn};
 
+/// The message expiry window (30 days, in seconds).
+const MESSAGE_EXPIRY_WINDOW: u64 = 30 * 24 * 60 * 60;
```

Comment:
It's honestly unfortunate rust alloc doesn't have a time module for these constants

Reply:
`core` does! Though, just a type. Also have to still use `from_secs`. Will update to that though.

Reply:
RIP https://github.com/rust-lang/rust/blob/master/library/core/src/time.rs#L27-L39